clang/lib/AST/RecordLayoutBuilder.cpp

```diff
@@ -818,7 +818,7 @@ class ItaniumRecordLayoutBuilder {
   void setSize(CharUnits NewSize) { Size = Context.toBits(NewSize); }
   void setSize(uint64_t NewSize) { Size = NewSize; }
 
-  CharUnits getAligment() const { return Alignment; }
+  CharUnits getAlignment() const { return Alignment; }
```

Comment:
Is that used at all then?

Reply:
It looks like an internal class so I think the answer is no

Reply:
Nope, just something I happened to notice when debugging something else. Not sure how well-received tiny prs like these are, but figured it was worth pointing out.